Rakefile

```diff
@@ -1184,9 +1186,10 @@ namespace :release do
     branch = "cocoapods-#{version}"
     message = "Upgrade CocoaPods to v#{version}"
 
-    sh "git clone https://github.com/caskroom/homebrew-cask.git homebrew_cask" unless Dir.exists? "homebrew_cask"
+    FileUtils.remove_dir("homebrew_cask") if Dir.exists? "homebrew_cask"
```

Comment:
started to have to keep track of branches etc, figured it was just easier to nuke it each time